pkg/rtc/participant.go

```diff
@@ -3344,13 +3344,14 @@ func (p *ParticipantImpl) MoveToRoom(params types.MoveToRoomParams) {
 
 	p.params.Logger.Infow("move participant to new room", "newRoomName", params.RoomName, "newID", params.ParticipantID)
```

Comment:
this should be the last one on old participant logger. Not sure if there are async paths which get logger from participant and if that would end up with new room/pID for pre-move participant.

Also, I guess need to pass in the `roomID` in `params` to set that in context. Have to check.

Reply:
`roomID` may not have resolved when move is executed. Have to figure out how to get that patched in.